hack/image-mirroring/mirror-images.sh

```diff
@@ -1,20 +0,0 @@
-#!/bin/bash
```

Comment:
We don't need to remove this script. I would prefer keeping it and changing all image mirroring jobs to use the script instead of inlining a bash code into the args.